.gitignore

```diff
@@ -72,4 +72,6 @@ dist/
 !.github/**
 
 # any log
-*.log
\ No newline at end of file
+*.log
+
+.DS_Store
```

Comment:
also gitignore test-worker/test/asset-compute/worker/build ?

Reply:
that `build` directory is already gitignored.

Reply:
https://github.com/adobe/asset-compute-sdk/pull/22/files/a6b9fd5d3f889765aa36734a64fe4bacecc4e6ea#diff-a084b794bc0759e7a6b77810e01874f2R28